pvl_hsu_soiling.m

```diff
@@ -0,0 +1,351 @@
+%     reached.
+%
+%   RainThresh is a scalar for the amount of rain, in mm, in a an accumulation
+%     period needed to clean the PV modules. In periods where the 
+%     accumulated rain meets or exceeds RainThresh, the panels are assumed 
+%     to be cleaned immediately after the accumulation period 
+%     [1] suggests a good RainThresh could be 1mm, but the time period is
+%     not specified. Rain accumulation period length can be adjusted in the
+%     optional input RainAccPeriod.
+%
+%   Tilt is a scalar or vector for the tilt of the PV panels. Changing tilt
+%     angles (e.g. in tracking cases) can be accomodated, and tilt angles
+%     are correlated with the entries in Time.
+%
+%   PM2_5 is the concentration of particulate matter with diamter less than
+%     2.5 microns, in g/m^3. Historical US concentration data sets may be
+%     found at https://aws.epa.gov/aqsweb/airdata/download_files.html. With
+%     file descriptions (for hourly data sets) at 
+%     https://aqs.epa.gov/aqsweb/airdata/FileFormats.html#_hourly_data_files
+%   
+%   PM10 is the concentration of particulate matter with diamter less than
+%     10 microns, in g/m^3. Historical US PM data sets found as described
+%     above.
+%
+%   ModelType is an optional input to the function to determine the 
+%     the model type to be used in the soiling model. A
+%     value of "1" indicates that the Variable Deposition Velocity model
+%     shall be used, a value of "2" indicates that the Fixed Settling
+%     Velocity model shall be used, and a value of "3" indicates that the
+%     Fixed Deposition Velocity model shall be used. [1] indicates that the
+%     Fixed Settling Velocity model performs best under a wide range of
+%     conditions, and thus "2" is the default ModelType if none is entered. 
+%     Validation efforts by Sandia National Laboratories
+%     confirms these findings. If an incorrect ModelType is provided, the
+%     Fixed Settling Velocity will be used (with a warning).
+%
+%   RainAccPeriod is an optional input that specifies the period, in hours
+%     over which to accumulate rainfall totals before checking against the
+%     rain cleaning threshold. For example, if the rain threshold is
+%     0.5 mm per hour, then RainThresh should be 0.5 and RainAccPeriod
+%     should be 1. If the threshold is 1 mm per hour, then the values
+%     should be 1 and 1, respectively. The minimum RainAccPeriod is 1
+%     hour. The default value is 1, indicating hourly rain accumulation.
+%     Accumulation periods exceeding 24 (daily accumulation) are not
+%     recommended.
+%   
+%   LUC is an optional input to the function, but it is required for the
+%     Variable Deposition Model. LUC is the Land Use Category as specified
+%     in Table 19.2 of [2]. LUC must be a numeric scalar with value 1, 4,
+%     6, 8, or 10, corresponding to land with evergreen trees, deciduous
+%     trees, grass, desert, or shrubs with interrupted woodlands. The LUC
```

Comment:
I'd delete the sentence about Brownian diffusion and the text about gamma (which isn't an input?).  It's not enough to explain the model, and adding more would make the variable definition too long.  If you want to include the material I'd make it comments in the function.

Reply:
Removed the two sentences about Brownian diffusion and gamma without adding comments in the function. Users should reference Seinfeld and Pandis for more information.